.github/pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+## Why was this change made? 🤔
+
+
+
+## How was this change tested? 🤨
+
+⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test) that use sdr-api*** (e.g.create_object_h2_spec.rb) and/or test in [stage|qa] environment, in addition to specs. ⚡
```

Comment:
uh - is this bit about sdr-api ... of relevance here?  I doubt it?

Reply:
Eek, I copied this template from globus_client, so will edit that in both places.